test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperListBoxExtensionsTest.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.TestCommon;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.Core
+{
+    public class HtmlHelperListBoxExtensionsTest
```

Comment:
Double-check that this file is the same as `HtmlHelperDropDownListExtensionsTest`, modulo the expected details and less the overloads `ListBox()` and `ListBoxFor()` don't support.